src/watchdog/src/watchdog.py

```diff
@@ -363,6 +375,9 @@ def main(args):
     parse_result = urlparse.urlparse(address)
     api_server_ip = parse_result.hostname
     api_server_port = parse_result.port or 80
+    tls = args.tls
+    ca_path = args.ca
+    bearer_path = args.bearer
```

Comment:
maybe should check if ca_path bearer_path is either both provided or not.

Reply:
check at each method

Reply:
checked

Reply:
Oh, I mean, you should check before entering the while loop of the main. That way the configuration error can be found before any logic began, it's easier for debug.

Reply:
checked